api/CacheStorage.json

```
@@ -398,6 +392,57 @@
             "deprecated": false
           }
         }
+      },
+      "worker_support": {
+        "__compat": {
+          "description": "Available in workers",
+          "support": {
+            "chrome": {
+              "version_added": "40",
+              "notes": "Version 40 to 42: supported in service workers only. Version 43 enabled use in main thread and other worker types."
```

I like how much clearer these notes and data are. A suggestion here to reframe this as consequential only for very old browsers:

```suggestion
              "notes": "Before version 43, only service workers are supported. From version 43, all worker types and the main thread are supported."
```

---

Thanks @ddbeck . I committed this. Slightly tempted by putting the current behaviour first because as you imply, the old behaviour isn't so interesting. So this would/could be. 

> From version 43, all worker types and the main thread are supported. Before version 43, only service workers were supported.